sdcm/nemesis.py

```diff
@@ -1744,6 +1756,14 @@ def wait_for_old_node_to_removed():
     def disrupt_kill_scylla(self):
         self._kill_scylla_daemon()
 
+    def disrupt_sigquit_scylla(self):
+        stress_cmd = self.tester.params.get('stress_cmd')
+        self.tester.run_stress_thread(
+            stress_cmd=stress_cmd, stress_num=1, stats_aggregate_cmds=False)
```

Comment:
nemesis rather don't run stress_cmd - unless it is by specific design (like doubling load)